docs/spec/v1/kustomization.md

```diff
@@ -0,0 +1,1750 @@
+# Kustomization
+
```

Comment:
We need to set a menu weight here as we did in [notification-controller](https://github.com/fluxcd/notification-controller/pull/498/files#diff-912aec2e27ddfd4a4a99a1c3882d3e9b5d6cdcae48d7566eaa09920c0327c432R3) so that the API reference navbar entry is properly rendered at the bottom.

Reply:
Done!